frontend/src/v5/ui/controls/assignees/assignees.component.tsx

```diff
@@ -42,7 +42,9 @@ export const Assignees = memo(({ assignees = [], max, className }: AssigneesType
 					<AssigneeListItem key={assignee} assignee={assignee} />
 				))
 			) : (
-				<FormattedMessage id="assignedAssignees.unassigned" defaultMessage="Unassigned" />
+				<UnassignedPlaceholder>
```

Comment:
Does it carry any problems if you just change the font  in `AssigneesList` in the style files ?